setup.py

```diff
@@ -38,7 +39,13 @@
         "zarr",
         "ome_zarr",
         "pymm_eventserver",
+        "ome-types[lxml]",
```

Comment:
note, as of v0.4.0, xmlschema is no longer a required dep of ome-types, so if you want to use the validate_xml method (which you do here) then you need to add either the `lxml` extra (recommended) or the `xmlschema` extra